include/zephyr/bluetooth/a2dp_codec_sbc.h

```diff
@@ -0,0 +1,121 @@
+/** @file
+ * @brief Advance Audio Distribution Profile - SBC Codec header.
+ */
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2015-2016 Intel Corporation
+ * Copyright (c) 2021 NXP
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
I'm not a legal expert, but it feels like it's a duplicate of SPDX-License-Identifier: Apache-2.0?

Reply:
I don't know which one is right, it is from original include/zephyr/bluetooth/a2dp-codec.h file.

Reply:
Can be updated in a future PR I think